tools/emscripten/pytest_scikit_image_pyodide.js

```diff
@@ -0,0 +1,57 @@
+// A JavaScript file to run the scikit-image test suite using Pyodide
+// This file is used by the GitHub Actions workflow to run the tests
+// against the Pyodide build of scikit-image defined in emscripten.yml.
+
+// The contents of this file are attributed to the scikit-learn developers,
+// who have a similar file in their repository:
+// https://github.com/scikit-learn/scikit-learn/blob/main/build_tools/azure/pytest-pyodide.js
```

Comment:
Could you update our [license](https://github.com/scikit-image/scikit-image/blob/main/LICENSE.txt) accordingly with

```
Files: tools/emscripten/pytest_scikit_image_pyodide.js
Copyright: 2007-2024 The scikit-learn developers
License: BSD-3-Clause
```

Reply:
Thanks for the heads up – I fixed this in 19159a4412309fc9310aa93458565a0ec035cc1d.

Reply:
Huh, the commit isn't shown as being part of this PR and also doesn't appear in the diff. Did you push that commit somewhere else accidentally?

Reply:
That's strange... I am able to see 19159a4412309fc9310aa93458565a0ec035cc1d (re-added the commit) below:

![image](https://github.com/scikit-image/scikit-image/assets/74401230/05786e49-8ff5-4850-bb1c-ea44a42e8b4c)

Here is the link to the commit that displays the diff from the PR's context: [`19159a4` (#7350)](https://github.com/scikit-image/scikit-image/pull/7350/commits/19159a4412309fc9310aa93458565a0ec035cc1d), maybe this helps?

Reply:
I can see it now. Probably me or my browser being confused. :thinking: Sorry.